client/src/components/forms/Login.js

```diff
@@ -1,16 +1,23 @@
 import React, { useState } from "react";
 import styled from "styled-components";
-import { Link } from "react-router-dom";
+import { Link, useHistory } from "react-router-dom";
 import { postData } from "../../helpers/fetch";
 
-function Login() {
+function Login({ setUserData, userData }) {
+  console.log(setUserData);
   const [input, setInput] = useState({});
+  const history = useHistory();
 
-  const handleSubmit = () => {
-    fetch("/login", {
+  const handleSubmit = async () => {
+    const apiCall = await fetch("/login", {
```

Comment:
`apiCall` isn't a very descriptive name. It doesn't tell much about what is specifically happening here. Naming stuff accurately is hard but what might be helpful is to always think of exactly what the function is doing. What API? What is the call doing? Is it fetching something? Updating/deleting something etc?

Better to have a long, descriptive name than a terse, vague one.